--- main.c	2019-01-08 22:30:38.859834774 -0500
+++ main.c.new	2019-01-08 22:30:28.000000000 -0500
@@ -68,6 +68,8 @@
 
 /* USER CODE BEGIN PFP */
 /* Private function prototypes -----------------------------------------------*/
+extern void setup();
+extern void loop();
 
 /* USER CODE END PFP */
 
@@ -103,7 +105,7 @@
   MX_USB_DEVICE_Init();
 
   /* USER CODE BEGIN 2 */
-
+  setup();
   /* USER CODE END 2 */
 
   /* Infinite loop */
@@ -113,7 +115,7 @@
   /* USER CODE END WHILE */
 
   /* USER CODE BEGIN 3 */
-
+    loop();
   }
   /* USER CODE END 3 */
 
